The-SMIRNOFF-force-field-format.md

```diff
@@ -1,103 +1,288 @@
    <Angle smirks="[#1:1]-[#6X4:2]-[#1:3]" angle="109.50" k="70.0"/>
+</AngleForce>     
 ```
 The first of these specifies the `[a,A:1]-[#6X4:2]-[a,A:3]` SMIRKS pattern for an angle, with a tetravalent carbon at the center with single bonds to two atoms of any type.
-Atoms are labeled 1, 2, and 3, with 2 being the central atom. Equilibrium angle values are provided, along with force constants (with units as given above).
+Atoms are labeled 1, 2, and 3, with 2 being the central atom.
+Equilibrium angle values are provided, along with force constants (with units as given above).
+
+Before getting in to individual sections, it's worth noting that the XML parser ignores attributes in the XML that it does not understand, so providing a parameter line for an angle that specifies (for example) a second force constant `k2` will lead to no effect.
 
 **SMIRNOFF parameters are hierarchical** in that parameters which come later in a file override those which come earlier if they match the same pattern.
  This can be seen in the example above, where the first line provides a generic angle parameter for any tetravalent carbon (single bond) angle, and the second line overrides this for the specific case of a hydrogen-(tetravalent carbon)-hydrogen angle.
 
 This hierarchical structure means that a typical parameter file will tend to have generic parameters early in the section for each force type, with more specialized parameters assigned later.
 
+Multiple `.ffxml` files can be loaded in sequence.
+If these files each contain unique `<...Force>` tags, the resulting forcefield will be independent of the order in which the files are loaded.
+If, however, the same tag occurs in multiple files, the contents of the tags are merged, with the tags read later appended to the *end* of the parameters read earlier, provided the `<...Force>` tags have compatible attributes.
+The resulting forcefield will therefore depend on the order in which parameters are read.
+This behavior is intended for limited use in appending very specific parameters (such as solvent models).
+
 **Technical note**: Because this is an XML format, certain special characters that occur in valid SMIRKS patterns (such as ampersand symbols `&`) must be treated specially to process.
+See [this entry](https://en.wikipedia.org/wiki/List_of_XML_and_HTML_character_entity_references) for more details.
+
+## Units
+
+If not specified by a `*_unit` attribute, the [standard OpenMM unit system](http://docs.openmm.org/latest/userguide/theory.html#units) is assumed:
+* distances are specified in `nanometers`
+* mass is specified in `daltons`
+* time is specified in `picoseconds`
+* charge is specified in `elementary_charge`,
+* temperature is specified in `kelvin`
+* quantity is specified in `moles`
+* angles are specified in `radians`
+
+**QUESTION:** Should we *require* the units instead of making them optional in order to minimize the possibility for mistakes?
```

Comment:
Requiring units: I'm ambivalent.

Reply:
I think we should require them but not a hill to die on.

Reply:
For now, I'll say specifying units is highly recommended.

Reply:
@davidlmobley : I just found a previous comment where you said they should be required. Which should we prefer here?